app/views/inventories/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1 class="text-center py-3">Inventories List</h1>
+
+<%= link_to "Add Inventory", new_inventory_path, class: "btn btn-primary mb-2 float-end" %>
+<% @inventories.each do |inventory| %>
+  <%= render partial: 'shared/inventory_card', locals: { inventory: inventory } %>
+<% end %>
```

Comment:
![image](https://github.com/GhausAliKhan/recipe-app/assets/93039359/3d40d534-73e9-4ebf-a91a-c401fa80a30c)

- [ ] Great work so far on inventory management but I see there is an issue while removing an inventory which has some food :fearful: When I try to remove an empty inventory then it is deleted :+1: but when I try to remove an inventory which has some foods in it then it shows me the inventory details page :thinking: Kindly fix this issue to make your app work professionally :+1: .